api/v1alpha08/kogitoserverlessworkflow_types.go

```diff
@@ -44,7 +111,7 @@ type ConfigMapWorkflowResource struct {
 // KogitoServerlessWorkflowSpec defines the desired state of KogitoServerlessWorkflow
 type KogitoServerlessWorkflowSpec struct {
 	// +kubebuilder:validation:Required
-	Flow model.Workflow `json:"flow"`
+	Flow Flow `json:"flow"`
```

Comment:
Is it better to put the `flow` as required?

Reply:
I don't follow, it's required:

```go
	// +kubebuilder:validation:Required

```

Reply:
yes you are right, I misread because I thought I saw a omitempty